client-v2/src/components/FrontsCAPIInterface/FeedItem.tsx

```diff
@@ -148,62 +142,57 @@ const getArticleLabel = (
   return startCase(sectionName);
 };
 
-const FeedItem = ({
-  id,
-  title,
-  href,
-  sectionName,
-  pillarId,
-  publicationDate,
-  internalPageCode,
-  firstPublicationDate,
-  isLive,
-  onAddToClipboard = noop,
-  scheduledPublicationDate,
-  tone,
-  thumbnail
-}: FeedItemProps) => (
+const FeedItem = ({ article, onAddToClipboard = noop }: FeedItemProps) => (
   <Container
     data-testid="feed-item"
     draggable={true}
-    onDragStart={event => dragStart(internalPageCode, event)}
+    onDragStart={event => dragStart(article, event)}
   >
     <VisitedWrapper
-      href={getPaths(id).live}
+      href={getPaths(article.id).live}
       onClick={e => e.preventDefault()}
       aria-disabled
     >
       <MetaContainer>
         <Tone
           style={{
             color:
-              getPillarColor(pillarId, isLive, tone === liveBlogTones.dead) ||
-              styleTheme.capiInterface.textLight
+              getPillarColor(
+                article.pillarId,
+                isLive(article),
+                article.frontsMeta.tone === liveBlogTones.dead
+              ) || styleTheme.capiInterface.textLight
           }}
         >
-          {getArticleLabel(firstPublicationDate, sectionName, isLive, tone)}
+          {getArticleLabel(article)}
         </Tone>
-        {scheduledPublicationDate && (
+        {article.fields.scheduledPublicationDate && (
           <ScheduledPublication>
             {distanceInWordsStrict(
-              new Date(scheduledPublicationDate),
+              new Date(article.fields.scheduledPublicationDate),
               Date.now()
             )}
           </ScheduledPublication>
         )}
-        {publicationDate && (
+        {article.webPublicationDate && (
```

Comment:
Presumably this is a just a correction you spotted during your travels 👍

Reply:
No actually - it's not that clear I did this - but because I needed the whole article inside a `FeedItem` now I just ditched all of the props and passed the `article` as a prop. Previously we were calling a `FeedItem` like `<FeedItem publicationDate={article.webPublicationDate} />`.

Reply:
Ah, gotcha.